setup.sh

```diff
@@ -5,20 +5,29 @@ then
 fi
```

Comment:
If no module is provided we should just stop.
```suggestion
    exit 1
fi
```